app/helpers/comments_helper.rb

```diff
@@ -2,4 +2,12 @@ module CommentsHelper
   def comment_as_html(text)
     Sanitize.clean(simple_format(text), Sanitize::Config::BASIC).html_safe
   end
+
+  def donation_declaration_notice
```

Comment:
I'd inline these. They're not reused and they don't contain much logic so it seems unnecessary to have them as helper methods.